.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,32 +1,23 @@
 ---
 name: Bug report
-about: Create a report to help us improve
+about: Report a bug or unexpected behavior
 ---
 
-**Describe the bug**
-A clear and concise description of what the bug is.
+**Summary**
 
-**To Reproduce**
-Reproduce it on stackblitz fork
-https://stackblitz.com/edit/cashmere-demo?file=app%2Fhome%2Fhome.component.ts
-_OR_
-Steps to reproduce the behavior:
+*Let us know what went wrong. The rest of this form is optional.*
 
-1.  Go to '...'
-2.  Click on '....'
-3.  Scroll down to '....'
-4.  See error
+**Reproduction**
 
-**Expected behavior**
-A clear and concise description of what you expected to happen.
+Steps to reproduce:
 
-**Screenshots**
-If applicable, add screenshots to help explain your problem.
+1. 
+2. 
+3. 
+4. 
 
-**Browser:**
-
--   Browser [e.g. chrome, safari]
--   Version [e.g. 22]
+Stackblitz reproduction (you can fork from the Stackblitz link in a component example):
```

Comment:
Great change! We were using the old stackblitz that I created to help people repro issues, but now we have the "edit in stack" button so it really iesn't needed.  You may want to be more explicit:

Stackblitz reproduction (optional). 
Click the "Edit in Stackblitz" button next to the most relevant sample and fork the generated angular template

Reply:
Updated, thanks for the suggestion.